paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -131,6 +140,69 @@ internal class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         disableViews()
     }
 
+    override fun onResume() {
+        super.onResume()
+        val sharedPreferences = getSharedPreferences(sharedPreferencesName, MODE_PRIVATE)
+
+        val customer = sharedPreferences.getString("customer", CheckoutCustomer.Guest.value)
+        val googlePay = sharedPreferences.getBoolean("googlePayConfig", true)
+        val currency = sharedPreferences.getString("currency", CheckoutCurrency.USD.value)
+        val mode = sharedPreferences.getString("mode", CheckoutMode.Payment.value)
+        val setShippingAddress = sharedPreferences.getBoolean("setShippingAddress", true)
+        val setAutomaticPaymentMethods = sharedPreferences.getBoolean("setAutomaticPaymentMethods", true)
+
+        setToggles(customer, googlePay, currency, mode, setShippingAddress, setAutomaticPaymentMethods)
+    }
+
+    override fun onPause() {
```

Comment:
I think a common use case will involve re-building and running the app on an emulator - does this still work in that scenario?

Reply:
Yep, tested it and it works